### ISSUE TITLE
Add "semantic tokens supported colorscheme" section

### ISSUE THREAD

**Issue by boltlessengineer** [rockerBOO/awesome-neovim#946]
It would be great to have special section for colorshcemes that support semantic tokens on top of _tree-sitter supported_ ones.

**Comment by rockerBOO** [comment]
Yeah we can probably expand this from only tree-sitter supported to include LSP highlighting including semantic tokens.

**Comment by cryptomilk** [comment]
nightcity.nvim support it ;-)

**Comment by DrKJeff16** [comment]
I'll close this issue since we have completed this task in #2029. Thanks for your heads-up!
